web/modules/custom/farm_nfa/src/PlanPermissionsRouteChecker.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\farm_nfa;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\user\Entity\Role;
+
+/**
+ * Checks access for displaying plan permissions route.
+ */
+class PlanPermissionsRouteChecker implements AccessInterface {
+
+  /**
+   * @param \Drupal\Core\Session\AccountInterface $account
```

Comment:
The description of the method is below, it should be above. You need to add a @param for $plan, and @return which I think is AccessResultInterface